developers/contributor-guide/current/weaviate-contributors-guide/git-and-github.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: layout-documentation
+solution: contributor-guide
+sub-menu: Weaviate Contributor's Guide
+title: Git and GitHub guide 
```

Comment:
I'm not sure this belongs in the contributor guide

Reply:
I believe it is still useful to provide guidance to those that are new to GitHub.
This way we can coach the new contributors enough to get started.

Reply:
My thinking here is that GitHub already has great resources regarding these topics. Maybe it would be better to link to existing documentation instead.

Such as:
- https://docs.github.com/en/get-started/quickstart/hello-world
- https://docs.github.com/en/get-started/quickstart/set-up-git
- https://docs.github.com/en/get-started/quickstart/github-flow
- https://docs.github.com/en/get-started/quickstart/contributing-to-projects

etc

Reply:
The problem that I see with providing detailed documentation for external entities is that we then assume the burden of keeping it up to date. If there are any upstream changes that we fail to notice, our documentation becomes inaccurate.

That being said, I'm not strongly opposed to including this.  But if we are keeping it there are some changes that need to be made. Comments incoming

Reply:
@Asmit2952 ok, I have finished reviewing the git/github guide. Comments added

Reply:
thanks for the review @parkerduckworth 
I've applied (almost) all your suggestions